### PR TITLE
docs: add example with styled components and scoping

### PR DESCRIPTION
**Pull request by lkuechler**
### Description

Add styled-components example with scoping of playgrounds enabled

### CODE REVIEW

File: examples/with-styled-components-and-scoping/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Docz Styled Components Example
+
+## Using `create-docz-app`
+
+```sh
+npx create-docz-app docz-app-styled-docz --example styled-components
```
Comment (rakannimer) [inline examples/with-styled-components-and-scoping/README.md]: styled-components -> with-styled-components-and-scoping

**Review by rakannimer — APPROVED** [review]
Minor nit in README.

File: examples/with-styled-components-and-scoping/README.md (hunk 1 above)
Comment (lkuechler) [reply examples/with-styled-components-and-scoping/README.md]: I hope i found them all now ^^

**Comment by rakannimer** [comment]
Awesome ! Thanks :)

**Pull request merged**